mobile/src/navigation/app/components/drawer-item/drawer-item.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { Pressable } from 'react-native';
+
+import { AppColor } from '~/common/enums/enums';
+import { DrawerNavigationItem } from '~/common/types/types';
+import { Text } from '~/components/common/common';
+import { useAppNavigate } from '~/hooks/hooks';
+
+import { styles } from './styles';
+
+const CustomDrawerItem: FC<DrawerNavigationItem> = ({
+  name,
+  iconName,
+  isFocused,
```

Comment:
On `DrawerNavigationItem` you've defined `isFocused` as optional, so maybe it would be better to provide default value for it